src/test/java/org/folio/rest/impl/CheckinReceivingApiTest.java

```diff
@@ -1120,8 +1120,8 @@ void testPostReceivingRevertElectronicResource() {
     assertThat(polSearches, not(nullValue()));
     assertThat(polUpdates, not(nullValue()));
 
-    // The piece searches should be made 3 times: 1st time to get piece record, 2nd and 3rd times to calculate expected PO Line status
-    assertThat(pieceSearches, hasSize(3));
+    // The piece searches should be made 2 times: 1st time to get piece record, 2nd times to calculate expected PO Line status
+    assertThat(pieceSearches, hasSize(2));
```

Comment:
We have 8 occurences of ReceivingStatus.EXPECTED in this class.
Let's change some of them to ReceivingStatus.CLAIM_DELAYED and ReceivingStatus.CLAIM_SEND to be safe if someone delete some status from list of available - the test will fail

Reply:
6 occurrences were related to updated condition and 4 of them changed with `CLAIM_DELAYED/CLAIM_SENT`